server/server.go

```diff
@@ -121,6 +126,14 @@ func (srv *server) setupRoutes() {
 	srv.r.HandleFunc(`/instances/{id}`, srv.handleInstanceByIDFetch).Methods("GET").Name("instance-by-id")
 	srv.r.HandleFunc(`/instances/{id}`, srv.handleInstanceByIDTerminate).Methods("DELETE").Name("instance-by-id-terminate")
 	srv.r.HandleFunc(`/instance-syncs`, srv.handleInstanceSync).Methods("POST").Name("instance-syncs-create")
+
+	if srv.enablePprof {
+		srv.r.HandleFunc(`/debug/pprof/`, pprof.Index).Name("pprof-index")
```

Comment:
I went for the same endpoints that Go uses by default here: https://github.com/golang/go/blob/4f7363cd723d3c79081d3a201279f4e024620db9/src/net/http/pprof/pprof.go#L67-L71